src/view.js

```diff
@@ -302,8 +302,8 @@ export default class View {
 
     // Set general uniforms that must be updated once per program
     setupProgram (program) {
-        program.uniform('2f', 'u_resolution', this.size.device.width, this.size.device.height);
-        program.uniform('3f', 'u_map_position', this.center.meters.x, this.center.meters.y, this.zoom);
+        program.uniform('2fv', 'u_resolution', [this.size.device.width, this.size.device.height]);
```

Comment:
Yeah if we want to avoid the variable-length arguments then we necessarily have to wrap the arguments in an array. The only only option I see is to have multiple `uniform` methods, one per type, such as `program.uniform2f(name, x, y)`.... but if you wanted to track the values in JS land you'd still need to create an array (or object with `x`, `y`, etc keys)